linux/scripts/upload-to-debian.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Upload the latest release to mentors.debian.net. Call from stable-* branch.
+# Usage: upload-to-debian.sh -k <Debian signing key>
+# Requires an entry in ~/.dput.cf:
+# ```
+# [mentors]
+# fqdn = mentors.debian.net
+# incoming = /upload
+# method = https
+# allow_unsigned_uploads = 0
+# progress_indicator = 2
+# allowed_distributions = .*
+# ```
+
+set -e
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+PROGRAM_NAME="$(basename "$0")"
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+error()
+{
+	echo -e "${RED}$PROGRAM_NAME: $1${NC}" >&2
+}
+
+log()
+{
+	echo -e "${GREEN}$PROGRAM_NAME: $1${NC}"
+}
+
+usage()
+{
+    echo ""
+    echo "Usage:"
+    echo "${0} -k <Debian signing key> [-n] [--push]"
+    echo ""
+    echo "Create source package for latest stable release and upload to Debian,"
+    echo "and update (and optionally push) changelog file."
+    echo ""
+    echo "    -k <key>    The PGP key used to sign the source package"
+    echo "    -n          Simulate upload"
+    echo "    --push      Push changelog changes to GitHub"
+    echo "    --help      Display usage help"
+}
+
+while (( $# )); do
+    case $1 in
+        -k) shift; debkeyid=$1;;
```

Comment:
```suggestion
        -k) shift; DEBKEYID=$1;;
```

I wonder if we should upcase this given our other vars are upper case?